plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/QLoginWebview.kt

```diff
@@ -161,19 +158,11 @@ class QWebviewBrowser(val project: Project, private val parentDisposable: Dispos
                 }
             }
 
-            "reauth" -> {
-                ToolkitConnectionManager.getInstance(project).activeConnectionForFeature(QConnection.getInstance())?.let { conn ->
-                    if (conn is ManagedBearerSsoConnection) {
-                        ApplicationManager.getApplication().executeOnPooledThread {
-                            reauthConnectionIfNeeded(project, conn, onPendingToken)
-                        }
-                    }
-                }
+            is BrowserMessage.Reauth -> {
+                reauth(ToolkitConnectionManager.getInstance(project).activeConnectionForFeature(QConnection.getInstance()))
             }
 
-            else -> {
-                error("received unknown command from Q browser: $command")
-            }
+            else -> {}
```

Comment:
if the type is sealed, do we still need this else branch?

Reply:
nah, removed it